src/EnergyPlus/WeatherManager.cc

```diff
@@ -281,6 +282,11 @@ namespace WeatherManager {
 	Array1D< SpecialDayData > SpecialDays;
 	Array1D< DataPeriodData > DataPeriods;
 
+	std::shared_ptr< BaseGroundTempsModel > siteShallowGroundTempsPtr;
```

Comment:
@Myoldmopar Yeah I see what he is doing now. You are right, it needs to be shared_ptr for this use case.